import-metadata.sh

```diff
@@ -1,43 +0,0 @@
-#!/bin/bash
-
```

Comment:
I'm not sure about this one. It's not necessary with lorax-composer, but the original plan was that lorax-composer is just an implementation backend for RHEL 7, and bdcs is the intended backend for the future. Did that change? AFAIK, bdcs still needed a metadata.db (although that is certainly a bug, too).